.github/workflows/build.yml

```diff
@@ -40,3 +35,61 @@ jobs:
           types: rpm
           maven-username: ${{ secrets.NEXUS_USERNAME }}
           maven-password: ${{ secrets.NEXUS_PASSWORD }}
+
+  build_centoslinux:
+    name: Centos7
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+      - name: Build
+        id: build
+        uses: ./.github/actions/centos-build
+      - uses: actions/upload-artifact@v3
+        with:
+          name: centos-rpms
+          path: rpms/*.rpm
+      - name: Deploy to nexus
+        uses: Alfresco/alfresco-build-tools/.github/actions/maven-deploy-file@v1.11.0
+        if: startsWith(github.ref_name, 'refs/tags/')
+        with:
+          group-id: org.imagemagick
+          artifact-id: imagemagick-distribution
+          repository-url: https://nexus.alfresco.com/nexus/content/repositories/thirdparty/
+          version: ${{ steps.build.outputs.built-version }}-ci
+          file: rpms/ImageMagick-${{ steps.build.outputs.built-version }}.x86_64.rpm
+          classifier: centos7
+          files: rpms/ImageMagick-libs-${{ steps.build.outputs.built-version }}.x86_64.rpm
+          classifiers: centos7-libs
+          types: rpm
+          maven-username: ${{ secrets.NEXUS_USERNAME }}
+          maven-password: ${{ secrets.NEXUS_PASSWORD }}
+
+  build_ubuntu:
+    name: Ubuntu-18.04
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+      - name: Build
+        id: build
+        uses: ./.github/actions/Ubuntu-build
+      - uses: actions/upload-artifact@v3
+        with:
+          name: ubuntu-deb
+          path: packages/*.deb
+      - name: Deploy to nexus
+        uses: Alfresco/alfresco-build-tools/.github/actions/maven-deploy-file@v1.11.0
+        if: startsWith(github.ref_name, 'refs/tags/')
+        with:
+          group-id: org.imagemagick
+          artifact-id: imagemagick-distribution
+          repository-url: https://nexus.alfresco.com/nexus/content/repositories/thirdparty/
+          version: ${{ steps.build.outputs.built-version }}-ci
+          file: rpms/imagemagick-7.1.0-16_${{ steps.build.outputs.built-version }}_amd64.deb
+          classifier: ubuntu
+          files: rpms/imagemagick-7.1.0-16-dev_${{ steps.build.outputs.built-version }}_amd64.deb
```

Comment:
path seems wrong (rpms instead of packages) and they should not have an hardcoded version